### DISCUSSION TITLE
Error with refreshTokenWithUser. (Please help me)

### DISCUSSION THREAD

**Discussion by oqildev** [saleor/saleor#17063] [Q&A]
I change fragments to our backend api structure. When I refresh browser, refreshTokenWithUser send old fragment data. I could'nt find problem. This is my code.
This is new Address structure:
export const fragmentAddress = gql`
  fragment Address on Address {
    id
    name
    city
    streetAddress
    houseNumber
    apartmentNumber
    houseSection
    doorCode
    note
    isDefaultShippingAddress
    isDefaultBillingAddress
  }
`;

This is old structure:
export const fragmentAddress = gql`
  fragment Address on Address {
    id
    firstName
    LastName
    companyName
    streetAddress
    streetAddress2
   .....
  }
`;

**Comment by tomaszlud** [comment]
Hello, I see you are trying to describe a problem with your application. Can you describe it in more detail, and the steps you take to trigger it? I'm curious to learn why you need to change that fragment. With further context, I should be able to help.

**Reply by oqildev** [reply]
I want to change the graphl fragment structure. According to our backend structure. Because, backend structure was changed.

I changed the graphl fragments according to the new structure. And I did "npm run fetch-schema". And I cleared the browser cache and restarted saleor. The site worked, but if I refresh the browser again or after a while, graphl sends a request to the api for the old fragment structure. Request name "refreshTokenWithUser".

 I changed all the file structures inside /fragments. And I did npm run fetch-schema and npm run dev. The problem was not solved.

When the browser refreshes the page, the old Saleor structure is requested again. 

![386144893-60ee507a-3b02-4d1e-925d-1b4d336df8d1](https://github.com/user-attachments/assets/ab3a7738-7a30-4a52-97e4-b373e25cd3b8)
![386144893-60ee507a-3b02-4d1e-925d-1b4d336df8d1](https://github.com/user-attachments/assets/38e70422-5e3f-41f4-aa44-38bb1e9306ac)

**Reply by oqildev** [reply]
This is request. 

![386144927-9286ac92-879e-4c5f-91c7-f9049177956b](https://github.com/user-attachments/assets/68fc014b-b925-4485-aca0-5e739cebb185)

**Reply by oqildev** [reply]
Why is the old AddressFragment structure being sent? I changed it. I can't find the codes for the request that is being sent.

**Comment by oqildev** [comment]
Please help me with this.
